addons/account/models/account_bank_statement.py

```diff
@@ -111,7 +111,7 @@ def _compute_date_index(self):
             stmt.first_line_index = sorted_lines[:1].internal_index
             stmt.date = sorted_lines.filtered(lambda l: l.state == 'posted')[-1:].date
 
-    @api.depends('create_date')
+    @api.depends('create_date', 'first_line_index')
```

Comment:
@ngochung207 I think it should be fixed like this
```suggestion
    @api.depends('create_date', 'line_ids.amount', 'line_ids.state')
```